client/settings/settings-manager/index.js

```diff
@@ -84,11 +84,11 @@ const DepositsDescription = () => {
 
 	return (
 		<>
-			<h2>{ __( 'Deposits', 'woocommerce-payments' ) }</h2>
+			<h2>{ __( 'Payouts', 'woocommerce-payments' ) }</h2>
 			<p>
 				{ sprintf(
 					__(
-						'Funds are available for deposit %s business days after they’re received.',
+						'Funds are available to be dispatched %s business days after they’re received.',
```

Comment:
Might be worth reviewing wording here, what are we trying to say? Maybe something like…

> Funds are dispatched to your payout account N business days after they're received.

or

> Funds are available for payout N business days after they're received.

Reply:
I went ahead with

> Funds are available for payout N business days after they're received.

[de6bd79](https://github.com/Automattic/woocommerce-payments/pull/9532/commits/de6bd7976f74190161a07d1213bdc8985435760c)